packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -38,6 +40,20 @@ function InstalledFonts() {
 	} = useContext( FontLibraryContext );
 	const [ isConfirmDeleteOpen, setIsConfirmDeleteOpen ] = useState( false );
 
+	const customFontFamilyId =
```

Comment:
Is it possible to include the capability check in `<Footer />`? (Genuine question, not a change request phrased as a question ;)

It's closer to the delete text in question and will prevent the UI showing no matter how the component is called.

Reply:
Looks like the footer component was removed in a recent update, so this is no longer an issue.